.changelog/64.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should we mention the change to the Dockerfile in the changelog?

Reply:
Yes! I don't know why I forgot to commit the release note message. Thanks! Updated.